Bottom: eeb8793013441fffc0a715de6c417c9962cdbdef
Top:    b934e80727956738d70af9d2c8138983137942ac
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 16:49:30 -0500

Use unprivileged ports for buildbot master server


---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 5fee835d..cfb0ce40 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -5,8 +5,9 @@
 buildbot_ip_addr: '127.0.0.1'
 buildbot_hostname: buildbot.example.com
 buildbot_port_args: 
-  - '80'
-  - '443'
+  - '8080'
+  - '8443'
+  - '9989'
 
 # Network names
 buildbot_networks:
@@ -39,7 +40,7 @@ buildbot_volumes:
 
 buildbot_env:
   BUILDBOT_DATABASE: 'sqlite:///{{ buildbot_data_path }}/state.sqlite'
-  BUILDBOT_WEB_PORT: '80'
+  BUILDBOT_WEB_PORT: '8080'
   BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
   BUILDBOT_SMTP_HOST: 'relay.dnsexit.com'
   BUILDBOT_SMTP_USER: 'Dean.cox'
